### PR TITLE
Set version in desired state for chart CRs

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/9079

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -44,9 +44,9 @@ required = [
   name = "github.com/docker/distribution"
   branch = "master"
 
-[[constraint]]
+[[override]]
```
Comment (rossf7) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 2]
```diff
@@ -44,9 +44,9 @@ required = [
...
   name = "github.com/giantswarm/apiextensions"
-  branch = "master"
+  branch = "chart-version"
```
Comment (rossf7) [inline Gopkg.toml]: TODO

**Comment by tomahawk28** [comment]
Using it for comparison would come to later PRs?

**Comment by rossf7** [comment]
> Using it for comparison would come to later PRs?

@tomahawk28 Sorry I got distracted when submitting last PR. Changes are used here https://github.com/giantswarm/chart-operator/pull/353

**Comment by rossf7** [comment]
Tested on `ghost`. Going with a ping.

**Pull request merged**